### ISSUE TITLE
[Bug] Tauon does not start after update.

### ISSUE THREAD

**Issue by keystroke3** [Taiko2k/TauonMusicBox#121]
I just updated tauon from the AUR and now it will not start. I decided to open it using the terminal and this is what I got:
![Screenshot from 2019-05-20 10-55-55](https://user-images.githubusercontent.com/37796862/58005432-51853400-7aee-11e9-8911-cb7afd7519d5.png)
I don't know what the gi is supposed to do, but it seems to be the problem. I tried to install it using pip but it does not seem to be there.

**Comment by Taiko2k** [comment]
Strange.

`gi` is provided by `python-gobject`, though it's often other issues that cause the import to fail. I'm not sure what that might be. Though you could try reinstalling that.

**Comment by keystroke3** [comment]
The issue still persists. I did a full system upgrade last night and did the AUR updates today.  The library seems to be existing, I reinstalled it.

**Comment by Taiko2k** [comment]
Just to test, if you open up the python interpreter `python` in a terminal and type the line `import gi`, does that give an error?

**Comment by keystroke3** [comment]
I get a ModuleNotFound error. Even though the library is installed.

**Comment by Taiko2k** [comment]
I just tried installing with Manjaro in a VM and it seems to be working. So I assume its a system package issue on your end, try double check your system is up to date and reboot maybe?

**Comment by keystroke3** [comment]
I have python installed in a virtual environment using` pyenv`. It seems it was not playing well with the `gi` , so the first thing I did was remove the `pyenv` and reinstalled it. I then uninstalled tauon and did a clean build using AUR. I had to install `discorgs_client` which the system did not have, apparently. I then changed the system python interpreter to the `pyenv` version. It works now.  
What I don't understand is why it failed to work yet everything was fine. All other packages that depend on python work well with the `pyenv` version even after the updates. Also, I don't know why I had to install `discorgs_client` yet I have never had to install it before.
